### PR TITLE
GHC-104: Add 'Procedure' to program_attribute_types

### PR DESCRIPTION
**Pull request by basrem**

### CODE REVIEW

File: configuration/concepts/program_attribute_types.csv [hunk 1]
```diff
@@ -22,16 +22,19 @@ bef52504-d77b-485c-ab81-9ccb6eb8f239,,,Pediatrics,Pediatrics,,,,,Misc,N/A,,,,,,,
 2484cdc7-a77a-4305-8bcc-b9c90f384bf6,,,Cardiology,Cardiology,,,,,Misc,N/A,,,,,,,,,,,
 dc04cfe1-29d1-46a7-a0cf-b8baacfdbb31,,,Dermatology,Dermatology,,,,,Misc,N/A,,,,,,,,,,,
 13aa8824-8a29-46c9-b80e-9201f03e82e5,,,Wound,Wound,,,,,Misc,N/A,,,,,,,,,,,
+d0e2b959-bd5d-4547-a545-d0e3e4b417de,,,Needs Clearance,Needs Clearance,,,,,Misc,N/A,,,,,,,,,,,
```
Comment (reagan-meant) [inline configuration/concepts/program_attribute_types.csv]: @basrem Is it necessary to add this here?

File: configuration/concepts/program_attribute_types.csv [hunk 2]
```diff
@@ -22,16 +22,19 @@ bef52504-d77b-485c-ab81-9ccb6eb8f239,,,Pediatrics,Pediatrics,,,,,Misc,N/A,,,,,,,
...
 589b55a5-40e9-4b7c-8af6-1415b2c6a0bf,,,Needs Surgical Optimization,Needs Surgical Optimization,,,,,Misc,N/A,,,,,,,,,,,
+ef83d1d9-d355-4356-9b60-61217f9dc730,,,Cleared for Surgery,Cleared for Surgery,,,,,Misc,N/A,,,,,,,,,,,
```
Comment (reagan-meant) [inline configuration/concepts/program_attribute_types.csv]: @basrem Is it necessary to add this here?
I see they are already in this [file](https://github.com/globalhealthcoalition/openmrs-config-ghc/blob/master/configuration/concepts/program_states.csv)

File: configuration/concepts/program_attribute_types.csv [hunk 3]
```diff
@@ -22,16 +22,19 @@ bef52504-d77b-485c-ab81-9ccb6eb8f239,,,Pediatrics,Pediatrics,,,,,Misc,N/A,,,,,,,
...
+0c401acb-5188-4bb5-8f6b-e5fbc28a6abf,,,Procedure,Procedure,,,,,Misc,Text,,,,,,,,,,,
+d47ce963-77fa-494a-ab37-7f0806d41ecb,,,Priority 1,Priority 1,,,,,Misc,Coded,,Upper extremity;Lower extremity;Orthopedic/Peds;Sports Orthopedics;Foot/ankle;Podiatry;Total Hip;Total Knee;Shoulder;Spine;Surgical Oncology;General Surgery;ENT;Urology;Gynecology;Ophthalmology;Dental;Procedure,,,,,,,,,
+c69ff88e-309b-41a0-99ae-5eb58dfc5966,,,Priority 2,Priority 2,,,,,Misc,Coded,,Upper extremity;Lower extremity;Orthopedic/Peds;Sports Orthopedics;Foot/ankle;Podiatry;Total Hip;Total Knee;Shoulder;Spine;Surgical Oncology;General Surgery;ENT;Urology;Gynecology;Ophthalmology;Dental;Procedure,,,,,,,,,
+8a6d5b95-ac55-41c2-b27e-ce18012bf962,,,Priority 3,Priority 3,,,,,Misc,Coded,,Upper extremity;Lower extremity;Orthopedic/Peds;Sports Orthopedics;Foot/ankle;Podiatry;Total Hip;Total Knee;Shoulder;Spine;Surgical Oncology;General Surgery;ENT;Urology;Gynecology;Ophthalmology;Dental;Procedure,,,,,,,,,
```
Comment (reagan-meant) [inline configuration/concepts/program_attribute_types.csv]: 'Procedure' is also an appointment_service_type and not an answer to the Priority 1,Priority 2,Priority 3...

**Review by reagan-meant — CHANGES_REQUESTED** [review]

File: configuration/concepts/program_attribute_types.csv (hunk 3 above)
Comment (basrem) [reply configuration/concepts/program_attribute_types.csv]: Done

**Comment by reagan-meant** [comment]
Closed in favour of [this](https://github.com/globalhealthcoalition/openmrs-config-ghc/pull/80)

**Pull request closed without merging**